plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -222,6 +222,13 @@ public void testCharVarcharComparison()
                 .hasMessage("Unsupported type: char(3)");
     }
 
+    @Test
+    @Override // Override because some tests (e.g. testKeyspaceNameAmbiguity) cause column listing failure
+    public void testShowInformationSchemaTables()
+    {
+        executeExclusively(super::testShowInformationSchemaTables);
```

Comment:
This test failed on master today: https://github.com/trinodb/trino/actions/runs/8822486117/job/24220825485